frontend/src/components/MapView.js

```diff
@@ -192,8 +205,11 @@ class MapView extends Component {
         imageUrl = imageUrl.split(",")[0];
       }
 
+
       return(
         <EventInfoWindow
+          onOpen={this.windowHasOpened}
+          onclose={this.windowHasClosed}
```

Comment:
I don't know if this is allowed, but the capitalization of these two lines are inconsistent